demo/full/styles/style.css

```diff
@@ -236,25 +271,39 @@ body {
 .controls-right-side {
   display: flex;
   justify-content: flex-end;
-  width: 100%;
+  align-items: center;
+  flex-grow: 1;
 }
 
-.load-button {
-  width: 70px;
-  border: none;
+.choice-input-button-wrapper {
+  margin-right: -5px;
+}
+
+.choice-input-button {
   font-family: "icons", sans-serif;
-  color: #222;
-  font-size: 1.5em;
-  cursor: pointer;
-  outline: 0;
-  background-color: #eee;
-  box-shadow: 0px 0px 2px 0px #fff;
-  border-radius: 3px;
+  font-size: 1.3rem;
+  border: solid 1px lightgray;
+  background-color: white;
+  margin: 5px;
+  height: 40px;
+  width: 60px;
+  background-color: #f9f9f9;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected duplicate "background-color" ([css:S4656](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4656))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXO-fuRlgbYnohYC6Zs&open=AWXO-fuRlgbYnohYC6Zs)

[ISSUE_KEY]: AWXO-fuRlgbYnohYC6Zs